python_examples/age_simulation.py

```diff
@@ -0,0 +1,93 @@
+#
+# Example simulation script with data output and visualisation
+#
+
+import os
+import logging
+from age_stratified_plot import Plotter
+import pyEpiabm as pe
+
+# Setup output for logging file
+logging.basicConfig(filename='sim.log', filemode='w+', level=logging.DEBUG,
+                    format=('%(asctime)s - %(name)s'
+                            + '- %(levelname)s - %(message)s'))
+
+# Set config file for Parameters
+dirname = os.path.dirname(os.path.abspath(__file__))
+paramfile = (dirname.split(os.sep))
+paramfile = paramfile[1:-1]
+paramfile.append("spatial_example/spatial_parameters.json")
+paramfile = os.path.join('C:\\', *paramfile)
```

Comment:
As previously discussed is this really needed? Please try the alternative I sent you @Elizabeth-Hayman - if it doesn't work we can debug it but this shouldn't be necessary